lib/chef/resource/cron_access.rb

```diff
@@ -76,9 +83,10 @@ class CronAccess < Chef::Resource
 
       action :deny do
         description "Add the user to the cron.deny file."
+        deny_path = File.join(value_for_platform_family(CRON_PATHS), "cron.deny")
```

Comment:
This is causing Test Kitchen failures here https://buildkite.com/chef-oss/chef-chef-master-verify/builds/4249#dc7f75c5-4bfe-4e17-8e2b-ea3ebce0ffd7/953-2471

I think you might want to change both of these to 
```suggestion
        deny_path = ::File.join(value_for_platform_family(CRON_PATHS), "cron.deny")
```

Also can you rebase this branch against master. There's a conflict on the description field now.